pygmt/src/tilemap.py

```diff
@@ -0,0 +1,145 @@
+"""
+tilemap - Plot XYZ tile maps.
+"""
+
+from pygmt.clib import Session
+from pygmt.helpers import (
+    GMTTempFile,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+
+try:
+    import rioxarray
+except ImportError:
+    rioxarray = None
+
+
+@fmt_docstring
+@use_alias(
+    A="img_out",
+    B="frame",
+    C="cmap",
+    D="img_in",
+    E="dpi",
+    G="bit_color",
+    I="shading",
+    J="projection",
+    M="monochrome",
+    N="no_clip",
+    Q="nan_transparent",
+    # R="region",
+    V="verbose",
+    n="interpolation",
+    c="panel",
+    f="coltypes",
+    p="perspective",
+    t="transparency",
+    x="cores",
+)
```

Comment:
I'm unsure about these aliases. I would like to keep a few parameters (e.g., `frame`, `projection`, `verbose`, `panel`, `transparency`. Does `cmap` work for images?) in the initial implementation and add more later.

Reply:
> I'm unsure about these aliases. I would like to keep a few parameters (e.g., `frame`, `projection`, `verbose`, `panel`, `transparency`, ...) in the initial implementation and add more later.

Fair enough, let me take a look at what's actually relevant to RGB images and trim down the list of aliases.

> Does `cmap` work for images?) 

I just tried using `fig.tilemap(region=[-180.0, 180.0, -90, 90], zoom=0, frame="afg", cmap="turbo")`, and it doesn't error, but the CPT isn't used either. Maybe best to remove `cmap` then.

Reply:
Ok, I've removed some aliases in c3156097bb512386b7f861b9e7f2b45a185831db.